lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -2636,6 +2636,40 @@ class ConvertAtenPoolingBaseOp : public OpConversionPattern<AtenOpT> {
     }
   }
 
+  // Checks the validity of pooling parameters and stores them in the respective
+  // vector.
+  LogicalResult
+  checkAndGetPoolingParameters(AtenOpT op, ConversionPatternRewriter &rewriter,
```

Comment:
This function does not work for the op `AtenAdaptiveAvgPool2dOp`, so it should not be in this base class.